web/src/beta/lib/core/Map/ClusteredLayers/index.tsx

```diff
@@ -65,7 +65,7 @@ export default function ClusteredLayers({
   const renderLayer = useCallback(
     (layer: Layer) => {
       const a = atomMap?.get(layer.id);
-      return !layer.id || !a ? null : (
+      return !layer.id || !a || !layer.visible ? null : (
```

Comment:
@keiya01 Should this be like this? Or should checking visible, and including/removing happen at the atomMap level?

Reply:
This should be handled in [here](https://github.com/reearth/reearth/blob/main/web/src/beta/lib/core/engines/Cesium/Feature/index.tsx#L146) or [here](https://github.com/reearth/reearth/blob/main/web/src/beta/lib/core/engines/Cesium/Feature/index.tsx#L172).

Reply:
@keiya01 if it in [here](https://github.com/reearth/reearth/blob/main/web/src/beta/lib/core/engines/Cesium/Feature/index.tsx#L172) I will need to re-render the component to update the map

Reply:
@keiya01 I try to put it here but the layer is not removed in map until I restart the page